res/css/views/rooms/_EventTile.pcss

```diff
@@ -483,20 +487,12 @@ $left-gutter: 64px;
     }
 
     &[data-layout="irc"] {
-        .mx_EventTile_line .mx_RedactedBody {
-            padding-left: 24px; /* 25px - 1px */
-
-            &::before {
-                left: var(--right-padding);
-            }
-        }
```

Comment:
How come this is no longer required? I follow the rest of the changes but can't quite see how this can just be removed, perhaps I'm missing something

Reply:
Nice catch, thanks. I'm checking why this was removed.

Reply:
Thanks for the detective work and the explanation